src/Xamarin.Android.Build.Tasks/Tasks/CompileToDalvik.cs

```diff
@@ -113,19 +120,19 @@ protected override string GenerateCommandLineCommands ()
 					foreach (var l in AlternativeJarFiles) {
 						var fullPath = Path.GetFullPath (l.ItemSpec);
 						Log.LogDebugMessage ($"    {fullPath}");
-						sw.WriteLine (fullPath);
+						sw.WriteLine (fullPath.Normalize (NormalizationForm.FormC));
```

Comment:
What normalization form, if any, does NTFS/Fat32 use to store filenames?

Apparently, [*no* normalization form is specified](https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file):

> There is no need to perform any Unicode normalization on path and file name strings for use by the Windows file I/O API functions because the file system treats path and file names as an opaque sequence of **WCHAR**s.

`AlternativeJarFiles` is coming from MSBuild/the file system, and thus *presumably* will be "correct" for file I/O purposes, in turn suggesting that we *shouldn't* normalize this filename, as such normalization may *prevent* the file from being found via e.g. `File.Open()` or whatever Java uses.